Content.Server/Materials/MaterialComponent.cs

```diff
@@ -22,6 +22,9 @@ public sealed class MaterialComponent : Component
         // ReSharper disable once CollectionNeverUpdated.Local
         private readonly List<string> _materials = new();
         public IEnumerable<string> MaterialIds => _materials;
+        
+        [DataField("volume")]
+        public int volume = 100;
 
```

Comment:
Based on the component's comment I don't think this should be here, also considering the fact that multiple materials can be stored here. At the very least it'd need to be a dictionary for materials.

@Zumorica

Reply:
I couldn't find an example of something using multiple material types at once, is that something that will be need to be supported?

Reply:
AFAIK this should still be done per material, probably as a dictionary.